bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/Layer/LayerTools.jsx

```diff
@@ -79,19 +79,17 @@ LayerTools.propTypes = {
 };
 
 const LayerStatus = ({ backendStatus, model, onClick }) => {
-    const icon = (
-        <LayerIcon
-            fill={backendStatus.color}
-            type={model.getLayerType()}
-            hasTimeseries={model.hasTimeseries()}
-            onClick={onClick ? () => onClick() : undefined}
-        />
-    );
-
-    return <Tooltip title={<Message messageKey={backendStatus.messageKey} />}>{icon}</Tooltip>;
+    return <LayerIcon
+        fill={ backendStatus.color }
+        type={ model.getLayerType() }
+        hasTimeseries={ model.hasTimeseries() }
+        onClick={ onClick ? () => onClick() : undefined }
+        backendStatus={ backendStatus.messageKey }
+    />;
 };
 
 LayerStatus.propTypes = {
+    layerType: PropTypes.string,
```

Comment:
I don't see LayerStatus handling this prop. It's taken from model and passed to LayerIcon?

Reply:
Removed this one. It is unused.